falsepositives.go

```diff
@@ -47,6 +47,7 @@ var falsePositives = []string{
 	"sassy",
 	"shoe",
 	"scuntthorpe",
+	"scunthorpe",
```

Comment:
```suggestion
	"scunthorpe",
```
Great catch! 
Might as well remove the wrong spelling while we're at it.

I believe there's a test in `goaway_test.go` that may need to be updated as well to reflect that.